src/test/ui/const-generics/issue-73298.rs

```diff
@@ -0,0 +1,24 @@
+// check-pass
```

Comment:
considering that this is a linker issue, don't we need `// build-pass` or `// run-pass` here?